src/omotes_simulator_core/entities/assets/asset_abstract.py

```diff
@@ -53,6 +46,8 @@ class AssetAbstract(ABC):
     """List of ids of the connected ports."""
     solver_asset: BaseAsset
```

Comment:
Kan je deze twee componenten, solver_asset en asset_type ook even voorzien van een docstring line?

Reply:
done

Reply:
Fix spacing to be the same as the above.